site/src/utils/time.ts

```diff
@@ -0,0 +1,31 @@
+export type TimeUnit = "days" | "hours";
+
+export function humanDuration(durationInMs: number) {
+  if (durationInMs === 0) {
+    return "0 hours";
```

Comment:
Would it be better to change this to return `"now"` when we have no time left?

I know that would mean that it could no longer slot in easily into the helper text component, but I feel like that return value would be more useful in more situations, especially for a general-purpose utility

Reply:
Hm... for this context I think "0 hours" is better than "now" but if in the future we see "now" is better we can update it. Sounds good?